src/_pytest/config/argparsing.py

```diff
@@ -29,7 +29,9 @@ def __init__(self, usage=None, processopt=None):
         self._groups = []  # type: List[OptionGroup]
         self._processopt = processopt
         self._usage = usage
-        self._inidict = {}  # type: Dict[str, Tuple[str, Optional[str], Any]]
+        self._inidict = (
```

Comment:
I think this have grown enough to deserve a `namedtuple` or `attrs` instance. What do you think?